egs2/wsj0_2mix/enh1/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+min_or_max=min # "min" or "max". This is to determine how the mixtures are generated in local/data.sh.
+sample_rate=8k
+
+
+train_set="tr_${min_or_max}_${sample_rate}"
+dev_set="cv_${min_or_max}_${sample_rate}"
```

Comment:
We decided to use the different variable names from `dev_set` and `eval_sets` in https://github.com/espnet/espnet/pull/2142
Please follow this way and modify here and `enh.sh` accordingly.

Reply:
It's done.